### ISSUE TITLE
exception occurred

### ISSUE THREAD

**Issue by eiryu** [johnlcox/toml4j#6]
```groovy
# groovy
println Toml.from('dob=1979-05-27T07:32:00-08:00').getAsString('dob')
```

```java
Caught: com.leacox.toml4j.ParseException: Invalid value: 1979-05-27T07:32:00-08:00
com.leacox.toml4j.ParseException: Invalid value: 1979-05-27T07:32:00-08:00
	at com.leacox.toml4j.TomlParser.parseValue(TomlParser.java:140)
	at com.leacox.toml4j.TomlParser.parse(TomlParser.java:88)
	at com.leacox.toml4j.TomlParser.parse(TomlParser.java:37)
	at com.leacox.toml4j.Toml.from(Toml.java:42)
	at com.leacox.toml4j.Toml$from.call(Unknown Source)
	at toml4j2.run(app.groovy:16)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:120)
```

**Comment by johnlcox** [comment]
It looks like timezone offsets are allowed in toml now. I should be able to fix this by just updating the date regex pattern.

Something like `^([\+-]?\d{4}(?!\d{2}\b))((-?)((0[1-9]|1[0-2])(\3([12]\d|0[1-9]|3[01]))?|W([0-4]\d|5[0-2])(-?[1-7])?|(00[1-9]|0[1-9]\d|[12]\d{2}|3([0-5]\d|6[1-6])))([T\s]((([01]\d|2[0-3])((:?)[0-5]\d)?|24\:?00)([\.,]\d+(?!:))?)?(\17[0-5]\d([\.,]\d+)?)?([zZ]|([\+-])([01]\d|2[0-3]):?([0-5]\d)?)?)?)?$`

**Comment by johnlcox** [comment]
Keeping the `Calendar` based API I will have to drop the timezone offset data from the input strings and return only UTC `Calendar` objects. This isn't ideal, so I'm considering pulling in joda-time for a better date API.

**Comment by johnlcox** [comment]
Fixed by fa01a6d5b63bb94d6c6e8ad3df93f0224ce71668
